src/robusta/integrations/discord/sender.py

```diff
@@ -38,6 +41,23 @@
 BLANK_CHAR = "\u200b"  # Discord does not allow us to send empty strings, so we use blank char instead
 
 
+class FindingStatus(Enum):
+    FIRING = 'firing'
+    RESOLVED = 'resolved'
+
+    def to_color_hex(self) -> str:
+        if self == FindingStatus.RESOLVED:
+            return "45826"
```

Comment:
This is really hex color?

Reply:
Discord only accepts decimal values of hex.

Reply:
created a new helper function `color_to_decimal`

Reply:
nice